src/transformers/image_utils.py

```diff
@@ -131,6 +131,13 @@ def convert_rgb(self, image):
 
         return image.convert("RGB")
 
+    def rescale(self, image: np.ndarray, scale: Union[float, int]) -> np.ndarray:
+        """
+        Rescale a numpy image by scale amount
+        """
+        self._ensure_format_supported(image)
```

Comment:
This checks whether the image is a PIL image, NumPy array or PyTorch tensor, but this method only expects NumPy arrays. So I'd update to raise a `ValueError` in case the `image` isn't an instance of np.ndarray.